src/tests/slot-composer-test.ts

```diff
@@ -115,7 +115,7 @@ recipe
     await slotComposer.expectationsCompleted();
   });
 
-  it('initialize recipe and render hosted slots', async () => {
+  it.skip('initialize recipe and render hosted slots', async () => {
```

Comment:
do we want to file an issue to investigate this failure?

Reply:
If it's being rewritten in a different way anyway, I don't see much point. It just times out intermittently.